src/experimental/lib/globals.ts

```diff
@@ -16,7 +16,7 @@ export type Callbacks = {
 
 declare global {
     // eslint-disable-next-line no-var
-    var _azleCallbacks: Callbacks;
+    var _azleCallbacks: Callbacks | undefined;
```

Comment:
src/experimental/build/commands/build/wasm_binary/rust/experimental_canister_template/src/execute_method_js.rs when we unwrap that will probably break if it's undefined... but I guess we don't care about that since it's experimental... for that matter that will probably apply to all of these, so why bother checking

Reply:
Yeah for experimental I say let's just not care, stable isn't much better but will return a Result::Err at least